### ISSUE TITLE
there is no elevation in this couponcard widget?

### ISSUE THREAD

**Issue by CKokila** [devmuaz/coupon_uikit#1]
for this i have tried to give boxshadow in decoration, but its not going outside of the card.

**Comment by devmuaz** [comment]
Hi @CKokila, Thanks for opening the issue.
I actually never tested this out with a `BoxShadow`, I will take a look as soon as I can and see why it's not applying Shadow.

You'll be notified once I make new related changes to it 🙏🏼

**Comment by devmuaz** [comment]
Hi @CKokila,
This issue has been fixed in this commit [#e09e390](https://github.com/devmuaz/coupon_uikit/commit/e09e390b19002202b9d8d2da7d96e62f3f67e769).

New property has been added in [v0.1.0](https://pub.dev/packages/coupon_uikit) named `shadow` which enables you to apply a shadow to the `CouponCard` widget like this:
```dart
CouponCard(
    shadow: BoxShadow(
        color: Colors.black56,
        blurRadius: 10,
    ),
),
```

I hope it does the job for you.
Enjoy!

**Comment by CKokila** [comment]
Now it's working. @devmuaz  thank you for your support.
